src/library/librarycontrol.cpp

```diff
@@ -211,6 +211,38 @@ LibraryControl::LibraryControl(Library* pLibrary)
             this,
             &LibraryControl::slotTrackColorNext);
 
+    // Control to navigate between widgets (tab/shit+tab button)
+    m_pSelectHistroyNext = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "search_history_next"));
+    m_pSelectHistoryPrev = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "search_history_prev"));
+    m_pSelectHistoryMove = std::make_unique<ControlEncoder>(
+            ConfigKey("[Library]", "search_history_move"), false);
```

Comment:
"move" is an odd word choice here. How about
```suggestion
            ConfigKey("[Library]", "search_history_selector"), false);
```
or "search_history_scroll"?